build/docker/lib/toybox_build.docker

```diff
@@ -3,7 +3,8 @@ ARG TOYBOX_SHA256="2bed6bb9edd5a249023103cf0402a835b0e53d10304a263f6f1e77a8aa49a
 
 ARG ROOT=/install_root
 
-RUN apt update && apt -y install musl musl-tools musl-dev
+RUN apt-get update && apt-get --no-install-recommends -y install musl=1.2.2-1 musl-tools=1.2.2-1 musl-dev=1.2.2-1
```

Comment:
I'd prefer we don't lock the versions even if the tool complains:
```suggestion
RUN apt-get update && apt-get --no-install-recommends -y install musl musl-tools musl-dev
```

Reply:
I can still see versions for all packages? Are you going to apply @mythi's suggestion?